environments/staging/inventory.ini

```diff
@@ -10,15 +10,16 @@
 # Amazon EC2
 proxy1
 
-[web0]
-10.201.10.104 swap_size=1G hostname=web0-staging ec2=yes
+# Temporarily commenting this out while it's removed from the rotation for the aws migration's couchdb2 reindex
+# [web0]
+# 10.201.10.104 swap_size=1G hostname=web0-staging ec2=yes
```

Comment:
I think you actually don't want to comment this part out, because you still want it to be under django_manage, right?

Reply:
Mostly for yourself

Reply:
https://github.com/dimagi/commcare-cloud/pull/2451/commits/3df39cc1324009d95847b02fd133325ab0bec8fa